tests/test_examples.py

```diff
@@ -208,6 +208,27 @@ def xfail_jit(*args):
                  marks=[requires_cuda]),
 ]
 
+FUNSOR_EXAMPLES = [
```

Comment:
Why not divvy these into `CPU_EXAMPLES`, `CUDA_EXAMPLES` and `JIT_EXAMPLES`? It looks like you're breaking `test_coverage`

Reply:
Alternatively you could update `test_coverage`